components/builder-web/app/packages-page/packages-page.component.ts

```diff
@@ -188,10 +188,7 @@ export class PackagesPageComponent implements OnInit, OnDestroy {
     private fetchBuilds() {
         if (this.origin && this.name) {
             let token = this.store.getState().gitHub.authToken;
-
-            if (token) {
-                this.store.dispatch(fetchBuilds(this.origin, this.name, token));
-            }
+            this.store.dispatch(fetchBuilds(this.origin, this.name, token));
```

Comment:
The server API for showing build jobs doesn't require authentication, which makes me think that passing a token to this function shouldn't need to happen.